similarity/config/test.configurations.properties

```diff
@@ -0,0 +1,20 @@
+memorybasedESA	src/test/resources/MemoryBasedESA_test.txt
+pageIDMapping	src/test/resources/wikiPageIDMapping_test.txt
+word2vec	src/main/resources/word2vec.txt
```

Comment:
I removed the word2vec line. The word similarity test case only test paragram. This configuration file is only used to test ESA and the sample ESA file in already included in test/resources.